server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -41,6 +42,24 @@ const (
 	actioncacheProtocolPrefix = "actioncache://"
 )
 
+var (
+	// AdminAPIMethods holds the list of service methods which are accessible
+	// only by group admins.
+	AdminAPIMethods = []string{
```

Comment:
FYI this is modeled as an explicit list of methods (rather than just checking `IsAdmin` in every method) so that we can return this to the client, so the client knows which RPCs it is authorized to perform without having to care about user roles.